### DISCUSSION TITLE
Node JS Configuration

### DISCUSSION THREAD

**Discussion by simibac** [decentralized-identity/veramo#478] [General]
I am trying to add some Veramo functionality to my node JS backend. However, the setup configuration from the node js tutorial seems outdated as described in [this issue here](https://github.com/uport-project/veramo/issues/450). 

Where can I find a configuration for `DIDResolverPlugin` that works with the latest version of Veramo?

**Comment by mirceanis** [comment]
Please see the comment in the issue and check if it works for you.
https://github.com/uport-project/veramo/issues/450#issuecomment-825623573

The next major release of veramo will fix this, but until then you can use the `@next` release channel.
